### ISSUE TITLE
Issue with %matplotlib inline in jupyter notebooks

### ISSUE THREAD

**Issue by dfm** [KeplerGO/lightkurve#861]
#### Problem description

In a few scenarios I'm finding that importing lightkurve in a jupyter notebook after running `%matplotlib inline` changes the matlotlib settings such that plots don't show up.

#### Example

A full example can be seen here: https://colab.research.google.com/drive/1SrqUIP5nN3_2GBxqphFLtak4ZoFV6tH-?usp=sharing but it's not isolated to Colab environments because I'm also getting it on GitHub actions.

Essentially: 

```python
%matplotlib inline
import lightkurve as lk
import matplotlib.pyplot as plt
plt.plot(...)
```

doesn't show a plot, but 

```python
import lightkurve as lk
%matplotlib inline
import matplotlib.pyplot as plt
plt.plot(...)
```

does.

#### Expected behavior

I would not expect importing lightkurve to affect the behavior of matplotlib.

#### Environment

-  platform (e.g. Linux, OSX, Windows): Ubuntu, Google Colab, perhaps others?
-  lightkurve version (e.g. 1.0b6): 1.11.2
-  installation method (e.g. pip, conda, source): pip

**Comment by barentsen** [comment]
The behavior is caused by this section of our `__init__.py`:

https://github.com/KeplerGO/lightkurve/blob/bb5e5d23a985a4cad3b3d4db6ce8d4c94d31f66a/lightkurve/__init__.py#L30-L36

I believe the reason for messing with the backend in this way was to make the plotting functions run out of the box on GitHub Actions and other headless Linux environments.  This is clearly a bad idea and should be fixed!

**Comment by barentsen** [comment]
Good news!  This issue has been fixed by #865 and is included in the new v1.11.3 and v2.0b3 releases.

I was not smart enough to come up with an automated unit test for this bug, but here is a screenshot to show that importing lightkurve no longer prohibits matplotlib plots from showing up in the Colab notebook you posted:

<img width="660" alt="Screen Shot 2020-10-06 at 20 00 12" src="https://user-images.githubusercontent.com/817669/95282559-99554a00-080e-11eb-9d92-8d854ef08435.png">


Thanks again for taking the effort to report this bug @dfm. I am grateful for your time! 🙏

**Comment by dfm** [comment]
Woohoo! Thanks!!
